README.md

```diff
@@ -78,6 +79,30 @@ module "dynamic_subnets" {
 }
 ```
 
+[`Geodesic`](https://github.com/cloudposse/geodesic) module usage example:
+
+
+add the terraform-aws-vpc module as a git submodule to the `conf/` directory in your geodessic module, set the git submodule to a specific branch or tag & update.
+
+```
+cd conf/
+
+git submodule init
+
+git submodule add https://github.com/cloudposse/terraform-aws-vpc.git terraform-aws-vpc
```

Comment:
This is where I'm not sold on submodules. In principle, I *want them to work*, but there's no way to override contents using strictly `git`. Using a hybrid of `git` and docker multi-stage, it is possible, but not sure if that's the best way either. 

For comparison, have you seen how terragrunt implements inheritance?